### PR TITLE
Added home page

### PR DESCRIPTION
**Pull request by Shrinkhlarajpoot**
Added All the Static Pages.
-HomePage
-Wishlist
-Cart
-Product
-Auth

### CODE REVIEW

File: src/frontend/pages/index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export {HomeScreen} from "./HomeScreen/HomeScreen"
+export {ProductScreen} from "./ProductScreen/ProductScreen"
+export {CartScreen} from "./CartScreen/CartScreen"
+export {WishlistScreen} from "./WishlistScreen/WishlistScreen"
+export {LoginScreen} from "./AuthScreen/LoginScreen"
+export {LogoutScreen} from "./AuthScreen/LogoutScreen"
+export {SignupScreen} from "./AuthScreen/SignupScreen"
+export {ForgetpassScreen} from "./AuthScreen/ForgetpassScreen"
+
```
Comment (pritam-kr) [inline src/frontend/pages/index.js:9]: It's good practice to use barrel export.

**Review by pritam-kr — APPROVED** [review]
Everything looks good to merge.
But you can host your images outside of the reppo.

File: src/frontend/components/Header/Header.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import "./Header.css";
+import {Link} from "react-router-dom";
+import { useState } from "react";
+const Header=()=>{
+  const[resnav,setResnav]=useState(false)
+
```
Comment (himadri2110) [inline src/frontend/components/Header/Header.js:6]: The naming here can be a bit more meaningful. `resnav` doesn't convey anything specific.

**Review by himadri2110 — APPROVED** [review]
A few suggestions: 

- The components' file should have a `.jsx` extension 
- When you start implementing functionality, remove the `screen` suffix from your files/folders
- You can use Prettier to format your files, it will improve readability

Nice work on breaking down the layout into small components 👍🏻

**Comment by Shrinkhlarajpoot** [comment]
> A few suggestions:
> 
> * The components' file should have a `.jsx` extension
> * When you start implementing functionality, remove the `screen` suffix from your files/folders
> * You can use Prettier to format your files, it will improve readability
> 
> Nice work on breaking down the layout into small components 👍🏻

Thanks for these suggestions..Will work on it

**Comment by Shrinkhlarajpoot** [comment]
> Everything looks good to merge. But you can host your images outside of the reppo.

okay Will do it..thanks

**Pull request merged**